cpp/src/parquet/encoding_test.cc

```diff
@@ -963,8 +963,6 @@ TYPED_TEST(EncodingAdHocTyped, ByteStreamSplitArrowDirectPut) {
 }
 
 TYPED_TEST(EncodingAdHocTyped, RleArrowDirectPut) {
-  // TODO: test with nulls once RleBooleanDecoder::DecodeArrow supports them
-  this->null_probability_ = 0;
```

Comment:
Do we have coverage yet?

Reply:
Hmm what does this mean? The case is covered here